src/types/types.ts

```diff
@@ -175,14 +175,17 @@ export enum Network {
   GEIST_POLTER = 'geist-polter',
   LUMIA_PRISM = 'lumia-prism',
   LUMIA_TESTNET = 'lumia-testnet',
+  UNICHAIN_MAINNET = 'unichain-mainnet',
   UNICHAIN_SEPOLIA = 'unichain-sepolia',
   SONIC_MAINNET = 'sonic-mainnet',
   SONIC_BLAZE = 'sonic-blaze',
   XMTP_TESTNET = 'xmtp-testnet',
+  ABSTRACT_MAINNET = 'abstract-mainnet',
   ABSTRACT_TESTNET = 'abstract-testnet',
   DEGEN_MAINNET = 'degen-mainnet',
   INK_MAINNET = 'ink-mainnet',
-  INK_SEPOLIA = 'ink-sepolia'
+  INK_SEPOLIA = 'ink-sepolia',
+  SEI_TESTNET = 'sei-testnet'
```

Comment:
are we adding INK as well in this PR? title doesn't mention it, just confirming.

Reply:
Otherwise looks good.

Reply:
Just a comma.